image/postgres/create-bundle.sh

```diff
@@ -30,8 +30,10 @@ chmod -R 755 "${bundle_root}"
 # =============================================================================
 # Get latest postgres minor version
 arch="x86_64"
+dnf_list_args=()
 if [[ $(uname -m) == "arm64" ]]; then
   arch="aarch64"
+  dnf_list_args=('--nogpgcheck')
```

Comment:
Can you mention the `package issue` you refer to in the PR description in a comment here? Also: because this is only applied to `dnf list`, I would expect that this is relatively benign. When installing the downloaded RPM in the image, we will do a GPG check, no?